controllers/module_reconciler.go

```diff
@@ -103,6 +103,7 @@ func NewModuleReconciler(
 //+kubebuilder:rbac:groups=apps,resources=daemonsets,verbs=create;delete;get;list;patch;watch
 //+kubebuilder:rbac:groups="core",resources=nodes,verbs=get;list;watch
 //+kubebuilder:rbac:groups="core",resources=secrets,verbs=get;list;watch
+//+kubebuilder:rbac:groups="core",resources=configmaps,verbs=get;list
```

Comment:
I though we also need to watch in case the user update the configMap before the image finished to build and was pushed.

Reply:
it is a corner case. We will get the ConfigMap update once the Build fails